samples/Sentry.Samples.Console.Basic/Program.cs

```diff
@@ -33,8 +34,37 @@
 
     // This option tells Sentry to capture 100% of traces. You still need to start transactions and spans.
     options.TracesSampleRate = 1.0;
+
+    options.EnableLogs = true;
+    options.SetBeforeSendLog(static (SentryLog log) =>
+    {
+        if (log.TryGetAttribute("plan.type", out string? attribute) && attribute == "enterprise")
+        {
+            return null;
+        }
+
+        return log.Level is SentryLogLevel.Error ? log : null;
+    });
 });
 
+var configureLog = static (SentryLog log) =>
+{
+    log.SetAttribute("string-attribute", "value");
+    log.SetAttribute("boolean-attribute", true);
+    log.SetAttribute("integer-attribute", long.MaxValue);
+    log.SetAttribute("double-attribute", double.MaxValue);
+};
+
+SentrySdk.Logger.LogTrace("Hello, World!", null, configureLog);
+SentrySdk.Logger.LogDebug("Hello, .NET!", null, configureLog);
+SentrySdk.Logger.LogInfo("Information", null, configureLog);
```

Comment:
the `null` in the public API isn't ideal, lets see if we can do something about that

Reply:
Will add appropriate overloads in a follow-up, to not bloat this one too much: https://github.com/getsentry/sentry-dotnet/issues/4132#issuecomment-2883482296